.github/workflows/repolinter.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v2
       - name: Lint Repo
+        continue-on-error: true
```

Comment:
Why not just keep the linter output in the console output of the action instead of an artefact?
It would enable checking the lint result code and get the output visible without the need to download and open the artefact because you can't view it online. Plus the output in the console is nicely coloured.

Reply:
Plus I would change the job name from "build" to something like "linting-repo" as it's may be confusing for users who think it's building Besu.